Error when attempting to play

Got the error below when trying to play.

Paper version:  1.21.5-78-main@d683970

 [dev.lavalink.youtube.http.YoutubeAccessTokenTracker] Updating YouTube visitor id succeeded, new one is CgtYX0dScUZtNU01MCjlx5rBBjIKCgJVUxIEGgAgLDoMCAEg8_DIytf8qJNoWP_S8dOp6vrF2AE%3D, next update will be after 600 seconds.
[02:33:10 WARN]: [dev.lavalink.youtube.cipher.SignatureCipher] Failed to find the tce variant sig function actions...
[02:33:10 ERROR]: [com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor] Error in playback of oA0CpI0vCK4
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
	at SimpleVoiceChatMusic-3.0.0.jar/com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:45) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:121) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.IllegalStateException: No match found
	at java.base/java.util.regex.Matcher.checkMatch(Matcher.java:1852) ~[?:?]
	at java.base/java.util.regex.Matcher.group(Matcher.java:687) ~[?:?]
	at SimpleVoiceChatMusic-3.0.0.jar/dev.lavalink.youtube.cipher.SignatureCipherManager.extractFromScript(SignatureCipherManager.java:338) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/dev.lavalink.youtube.cipher.SignatureCipherManager.getCipherScript(SignatureCipherManager.java:267) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/dev.lavalink.youtube.clients.skeleton.NonMusicClient.loadTrackInfoFromInnertube(NonMusicClient.java:110) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/dev.lavalink.youtube.clients.skeleton.StreamingNonMusicClient.loadFormats(StreamingNonMusicClient.java:35) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/dev.lavalink.youtube.track.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:208) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:145) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:93) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	at SimpleVoiceChatMusic-3.0.0.jar/com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109) ~[SimpleVoiceChatMusic-3.0.0.jar:?]
	... 4 more
	Suppressed: dev.lavalink.youtube.ClientInformation: 
  yts.version: 1.13.0
  client.identifier: ANDROID_VR
  client.options: ClientOptions{playback=true, playlistLoading=true, videoLoading=true, searching=true}
[02:33:10 ERROR]: [SimpleVoiceChatMusic] Failed to play 「MV」mumei due to error: {}

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,7 @@
 
     <groupId>dev.klash</groupId>
     <artifactId>SimpleVoiceChatMusic</artifactId>
-    <version>3.0.0</version>
+    <version>3.0.1</version>
     <packaging>jar</packaging>
 
     <name>SimpleVoiceChatMusic</name>
@@ -94,7 +94,7 @@
         <dependency>
             <groupId>dev.lavalink.youtube</groupId>
             <artifactId>v2</artifactId>
-            <version>1.13.0</version>
+            <version>1.13.2</version>
         </dependency>
         <dependency>
             <groupId>org.jsoup</groupId>

